### PR TITLE
add more readme, update links

### LINKED ISSUE #2: Update readme to better explain filter behavior
I think it would be beneficial to include an explanation of the filter behavior in the readme rather than linking out to the analog code in Sensu Core.

Specifically, I find users are sometimes surprised that OK events are allowed through, even though we know this is working-as-intended. Adding this information to the readme instead of providing it as an external link will reduce surprises.

### PR DESCRIPTION
**Pull request by cwjohnston**
Closes #2

### CODE REVIEW

**Comment by cwjohnston** [comment]
@hillaryfraley could use your 👀  please?

File: README.md [hunk 1]
```diff
@@ -8,9 +8,24 @@ critical, unknown).
 This Filter Asset implements the [Sensu 1.x built-in Handler
 severities filter logic][2].
 
-## Configuration
+The logic excludes events unless their check exit status matches
+the specified severity, excepting of certain "ok" severity events.
```
Comment (hillaryfraley) [inline README.md:12]:
```suggestion
The severities filter logic excludes events unless their check exit status matches
the specified severity, except for certain "ok" severity events. 
```

File: README.md [hunk 2]
```diff
@@ -8,9 +8,24 @@ critical, unknown).
 This Filter Asset implements the [Sensu 1.x built-in Handler
 severities filter logic][2].
 
-## Configuration
+The logic excludes events unless their check exit status matches
+the specified severity, excepting of certain "ok" severity events. 
 
-### Configure filter definitions
+OK events are allowed when the available check history includes
+a match for the specified severity.
```
Comment (hillaryfraley) [inline README.md:15]:
```suggestion
OK-severity events are allowed when the available check history includes
a match for the specified severity.
```

**Comment by hillaryfraley** [comment]
@cwjohnston just a couple suggestions

**Pull request merged**